contracts/apps/TicTacToe.sol

```diff
@@ -0,0 +1,163 @@
+pragma solidity 0.4.24;
+pragma experimental "ABIEncoderV2";
+
+import "../lib/Transfer.sol";
+
+
+contract TicTacToe {
+
+  enum ActionType {
+    PLAY,
+    PLAY_AND_WIN,
+    PLAY_AND_DRAW,
+    DRAW
+  }
+
+  enum WinClaimType {
+    COL,
+    ROW,
+    DIAG,
+    CROSS_DIAG
+  }
+
+  struct WinClaim {
+    WinClaimType winClaimType;
+    uint256 winClaimIdx;
```

Comment:
the "winClaim" part of `winClaimIdx` is redundant, just call it `idx` or `index` or `startIndex`.